Instructions/ReadMe.md

```diff
@@ -34,4 +51,11 @@ the beamline with respect to the sample using the mcstas `AT (x, y, z) RELATIVE
 where z is along the beam direction. Once can easily modify this position by changing the `z` value.
 The TOF detector is currently set to detect time of flight values between 0 - 0.1seconds (tmin and tmax).
 The Lambda detector is set to detect wavelengths between l_min_source and l_max_source which are
-calculated based on the instrument parameters lmin=3.0AA and lmax=9.7AA.
\ No newline at end of file
+calculated based on the instrument parameters lmin=3.0AA and lmax=9.7AA.
+
+
+### MCSTAS Team
+
+For further information contact the ESS Mcstas Team:
+    - Torben Nielsen (Torben.Nielsen@esss.se)
```

Comment:
Otherwise the text changes look good. I'll see what they look like in PDF form too.